### ISSUE TITLE
Low user interface visibility

### ISSUE THREAD

**Issue by art-rasa** [Nokse22/ascii-draw#17]
**Describe the bug:**
The canvas and "Styles" toolbox items have very low visibility and are almost invisible.

**Steps To Reproduce:**
1. Open ASCII Draw
2. Select some drawing tool, e.g. rectangle
3. Draw a rectangle on canvas

**Expected behaviour:**
- Symbols on canvas are expected to be visible after drawing
- Styles toolbox items are expected to be visible

**Screenshots if needed:**
https://imgur.com/a/0upNqpe

**System:**
 - App version: 0.2.0
 - Operating system: Fedora Linux 37

**Additional context:**
Light Gnome UI style

**Comment by Nokse22** [comment]
Hello. Since you are using Fedora 37 I'm guessing you have Gnome 44, but in the screenshot you provided the app doesn't use the default Adwaita theme, it looks more like a Gtk3 theme.
Can you tell me what theme are you using? Can you try to use the app without that theme?

**Comment by art-rasa** [comment]
I upgraded my OS to Fedora 38 but still having the same problem. I am using the default themes, _except_ I have added the following line on my ~/.profile file:

`export GTK_THEME=Adwaita:light`

I had to add it because I am unable to use any kind of "dark" themes due to my eyesight. Many Gnome software (such as GThumb) now use dark themes by default that cannot be changed in settings. Hope this helps, thanks.
